ONSdigital/rm-case-service
# RAS-1030 Migrate from the spring cloud pubsub providers to google cloud Case service

# What and why?
This PR migrates to the google cloud provided starter for pub sub since the old spring cloud ones are no longer maintained

# How to test?
- deploy this PR to you envrionment
- run the acceptance tests
- complete the survey in frontstage
- make sure that all the relevant pub sub messages are consumed and published properly by the case service

# Jira
https://jira.ons.gov.uk/browse/RAS-1030
